autoprotocol/protocol.py

```diff
@@ -6842,11 +6871,16 @@ def location_helper(aliquot, volume, method):
             if one_tip is True:
                 locations += location_helper(wel, vol, met)
             else:
+                location_transports = location_helper(wel, vol, met)
+                source_transports = location_transports[0]["transports"]
+                instruction_mode = mode
```

Comment:
I'm confused why you need to assign mode to a new object named `instruction_mode`. Can we just leave this as `mode` and refer to it as such below?

Reply:
It is there because this happens inside a `for` loop. i.e. if I just call it `mode` it will overwrite the input `mode` the second time it goes through loop.